src/Shell/SphinxConfShell.php

```diff
@@ -289,6 +289,8 @@ class SphinxConfShell extends Shell {
         'U+a800..U+a827',
         # Ol Chiki, for Santali (sat)
         'U+1c50..U+1c7d',
+        # Hanifi Rohingya
+        'U+10D00..U+10D23', 'U+10D30..U+10D39'
```

Comment:
Are you sure the first range shouldn't go all the way to U+10D27? The characters you left out are diacritics, but that doesn't necessarily make them particularly ignorable. (Also, ignoring them this way treats them like spaces, which is probably as undesirable as splitting French words after every character with an accent mark.)

Reply:
No, I wasn't sure (hence why I left the branch unmerged). I didn't know exactly what effect it would have to include or exclude these characters.

I updated the charset range during the [last Tatoeba stream](https://youtu.be/3aRkxkbus_A?t=1415), following your and gillux's recommendation.